public/sass/pages/_explore.scss

```diff
@@ -55,6 +55,25 @@
     margin-top: 2 * $panel-margin;
   }
 
+  .time-series-disclaimer {
+    width: 300px;
+    margin: $panel-margin auto;
+    padding: 10px 0;
```

Comment:
No variable made sense here?

Reply:
Not to me, felt weird to use a margin-variable on padding. And the padding variable wouldn't match.